packages/docusaurus-plugin-ideal-image/src/theme/IdealImage.tsx

```diff
@@ -4,11 +4,19 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  */
+
 import React from 'react';
 import IdealImage from '@endiliey/react-ideal-image';
+import type {Props} from '@theme/IdealImage';
 
-function Image(props) {
+function Image(props: Props): JSX.Element {
   const {alt, className, img} = props;
+
+  // In dev env just use regular img with original file
+  if (img.default) {
```

Comment:
I'm not sure it's a very reliable test. 
In the future (Webpack 5 asset pipeline, `new URL()` etc...) it may be possible that both won't have a default (but we can probably improve this test later anyway)

Reply:
Speaking of which, what is the progress in #4708?

Reply:
Standby for now, not a top priority atm but I'd like to make this work someday 😅